src/EnergyPlus/DataAirSystems.hh

```diff
@@ -214,204 +189,146 @@ namespace DataAirSystems {
+        int PlantLoopType = 0;
+        int PlantLoopNum = 0;
+        int PlantLoopBranch = 0;
+        int PlantLoopComp = 0;
+        int FirstDemandSidePtr = 0;
+        int LastDemandSidePtr = 0;
     };
 
     struct ConnectAirSysComp
     {
         // Members
-        int AirLoopNum;
-        int AirLoopBranch;
-        int AirLoopComp;
-        int PlantLoopType;
-        int PlantLoopNum;
-        int PlantLoopBranch;
-        int PlantLoopComp;
-        int FirstDemandSidePtr;
-        int LastDemandSidePtr;
-
-        // Default Constructor
-        ConnectAirSysComp()
-            : AirLoopNum(0), AirLoopBranch(0), AirLoopComp(0), PlantLoopType(0), PlantLoopNum(0), PlantLoopBranch(0), PlantLoopComp(0),
-              FirstDemandSidePtr(0), LastDemandSidePtr(0)
-        {
-        }
+        int AirLoopNum = 0;
+        int AirLoopBranch = 0;
+        int AirLoopComp = 0;
+        int PlantLoopType = 0;
+        int PlantLoopNum = 0;
+        int PlantLoopBranch = 0;
+        int PlantLoopComp = 0;
+        int FirstDemandSidePtr = 0;
+        int LastDemandSidePtr = 0;
     };
 
     struct ConnectAirSysSubComp
     {
         // Members
-        int AirLoopNum;
-        int AirLoopBranch;
-        int AirLoopComp;
-        int AirLoopSubComp;
-        int PlantLoopType;
-        int PlantLoopNum;
-        int PlantLoopBranch;
-        int PlantLoopComp;
-        int FirstDemandSidePtr;
-        int LastDemandSidePtr;
-
-        // Default Constructor
-        ConnectAirSysSubComp()
-            : AirLoopNum(0), AirLoopBranch(0), AirLoopComp(0), AirLoopSubComp(0), PlantLoopType(0), PlantLoopNum(0), PlantLoopBranch(0),
-              PlantLoopComp(0), FirstDemandSidePtr(0), LastDemandSidePtr(0)
-        {
-        }
+        int AirLoopNum = 0;
+        int AirLoopBranch = 0;
+        int AirLoopComp = 0;
+        int AirLoopSubComp = 0;
+        int PlantLoopType = 0;
+        int PlantLoopNum = 0;
+        int PlantLoopBranch = 0;
+        int PlantLoopComp = 0;
+        int FirstDemandSidePtr = 0;
+        int LastDemandSidePtr = 0;
     };
 
     struct ConnectAirSysSubSubComp
     {
         // Members
-        int AirLoopNum;
-        int AirLoopBranch;
-        int AirLoopComp;
-        int AirLoopSubComp;
-        int AirLoopSubSubComp;
-        int PlantLoopType;
-        int PlantLoopNum;
-        int PlantLoopBranch;
-        int PlantLoopComp;
-        int FirstDemandSidePtr;
-        int LastDemandSidePtr;
-
-        // Default Constructor
-        ConnectAirSysSubSubComp()
-            : AirLoopNum(0), AirLoopBranch(0), AirLoopComp(0), AirLoopSubComp(0), AirLoopSubSubComp(0), PlantLoopType(0), PlantLoopNum(0),
-              PlantLoopBranch(0), PlantLoopComp(0), FirstDemandSidePtr(0), LastDemandSidePtr(0)
-        {
-        }
+        int AirLoopNum = 0;
+        int AirLoopBranch = 0;
+        int AirLoopComp = 0;
+        int AirLoopSubComp = 0;
+        int AirLoopSubSubComp = 0;
+        int PlantLoopType = 0;
```

Comment:
:pushpin:. Also, each of these `struct` is a subtype of the previously defined `struct` with one added field. Just sayin`.

Reply:
https://github.com/NREL/EnergyPlus/issues/9372